docs/data/sdks/android-kotlin-ampli.md

```diff
@@ -0,0 +1,452 @@
+    	}
+
+    	@Override
+    	public BaseEvent track(BaseEvent event) {
+        	Properties properties = new Properties();
+        	for (Map.Entry<String,Object> entry : event.getEventProperties().entrySet()) {
+            	properties.putValue(entry.getKey(),entry.getValue());
+        	}
+        	analytics.track(event.eventType, properties);
+        	return event;
+    	}
+	}
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    class SegmentDestinationPlugin(appContext: Context, segmentApiKey: String) : DestinationPlugin() {
+    	var analytics: Analytics? = null;
+    	val context: Context = appContext;
+    	init {
+        	analytics = Analytics.Builder(appContext, segmentApiKey).build()
+    	}
+
+    	override fun track(event: BaseEvent): BaseEvent {
+        	val eventProperties =  Properties();
+        	event.eventProperties?.forEach { entry -> entry.value?.let {
+            	eventProperties.put(entry.key,
+                it)
+        	} }
+
+        	analytics?.track(event.eventType, eventProperties);
+        	return event
+    	}
+    }
+    ```
+
+Add your plugin after init Ampli.
+
+=== "Java"
+
+    ```java
+	Ampli.getInstance().getClient().add(
+		new YourDestinationPlugin(this, DESTINATION_API_KEY)
+	);
+
+    ```
+
+=== "Kotlin"
+
+    ```kotlin
+    ampli.client?.add(
+		YourDestinationPlugin(this, DESTINATION_API_KEY)
+	)
+    ```
+
+##  Verify implementation status
+
+Verify that events are implemented in your code with the status command:
+
+```bash
+ampli status
+```
+
+To update the implementation status in your tracking plan use the `--update` flag or `-u`:
+
+```bash
+ampli status -u
+```
+The output displays status and indicates what events are missing.
+
+```bash
+➜ ampli status
+✘ Verifying event tracking implementation in source code
+ ✔ Song Played (1 location)
+ ✘ Song Stopped Called when a user stops playing a song.
+Events Tracked: 2 missed, 3 total
+```
+
+Learn more about [`ampli status`](https://developers.data.amplitude.com/using-the-ampli-cli/#ampli-status).
+
+
+## Migrating from an Itly Android runtime
+
+Migrate from an Itly Android runtime to Ampli by following these steps.
+
+1. Remove legacy Itly dependencies from your project. This includes anything with a `ly.iterative.itly`.
+
+    ```bash
+    implementation "ly.iterative.itly:sdk-android:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-iteratively:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-schema-validator:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-amplitude-android:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-mixpanel-android:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-mparticle-android:$itlySdkVersion"
+    implementation "ly.iterative.itly:plugin-segment-android:$itlySdkVersion"
+    ```
+
+2. Add Amplitude dependencies.
+
+    ```bash
+    implementation 'com.amplitude:analytics-android:0.1.0-beta.9'
```

Comment:
Should this be some POST BETA version after the GA?

Reply:
Yes, need to update after kotlin GA.